jevents/jevents.c

```diff
@@ -58,7 +63,9 @@ static const char *json_default_name(char *type)
 			goto out;
 	}
 
-	cache = getenv("XDG_CACHE_HOME");
+	cache = getenv_copy("JEVENTS_CACHEDIR");
+	if (!cache)
+		cache = getenv_copy("XDG_CACHE_HOME");
```

Comment:
Can't free stuff from `getevn`, but we did below because we don't distinguish between this and the $HOME case (where `cache` does end up with a freeable string).

Easy solution is to `strdup` any string from getenv, so `cache` is always freeable.